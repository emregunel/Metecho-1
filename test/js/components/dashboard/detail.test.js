import { render } from '@testing-library/react';
import React from 'react';
import { StaticRouter } from 'react-router-dom';

import DashboardDetail from '@/js/components/dashboard/detail';

describe('<DashboardDetail/>', () => {
  const context = {};
  const { container } = render(
    <StaticRouter context={context}>
      <DashboardDetail match={{ path: '/dashboard' }} />
    </StaticRouter>,
  );

  test('renders dashboard', () => {
    expect(container).toMatchSnapshot();
  });
});
