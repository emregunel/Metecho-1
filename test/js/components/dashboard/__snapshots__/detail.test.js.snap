// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DashboardDetail/> renders dashboard 1`] = `
<div>
  <div
    class="dashboard-view"
  >
    <div
      class="slds-brand-band slds-brand-band_medium slds-brand-band_user dashboard-header"
    />
    <div
      class="slds-grid dashboard-profile slds-p-around_large slds-m-horizontal_xx-large"
    >
      <div
        class="slds-grid dashboard-stats"
      >
        <div
          class="slds-grid dashboard-stat-item"
        >
          svg
          <span
            class="dashboard-stat-highlight"
          >
            5
          </span>
          <span
            class="dashboard-stat-label"
          >
            Completed Epics
          </span>
        </div>
        <div
          class="slds-grid dashboard-stat-item"
        >
          svg
          <span
            class="dashboard-stat-highlight"
          >
            12
          </span>
          <span
            class="dashboard-stat-label"
          >
            Completed Tasks
          </span>
        </div>
        <div
          class="slds-grid dashboard-stat-item"
        >
          svg
          <span
            class="dashboard-stat-highlight"
          >
            20
          </span>
          <span
            class="dashboard-stat-label"
          >
            Epic Collaborations
          </span>
        </div>
      </div>
    </div>
  </div>
</div>
`;
